File: Blind75/Binary Tree Level Order Traversal/Binary_Tree_Level_Order_Traversal.py
def binaryTreeLevelOrderTraversal(root):
    res = []
    def backtrack(curr_node, level):
        if not curr_node: return
        if len(res) == level: res.append([])
        
        res[level].append(curr_node.val)
        backtrack(curr_node.left, level+1)
        backtrack(curr_node.right, level+1)
                        
    backtrack(root,0)
    return res

File: Blind75/Binary Tree Level Order Traversal/Binary_Tree_Level_Order_Traversal.js
function binaryTreeLevelOrderTraversal(root){
    let res = []
    function backtrack(currNode, level){
        if (!currNode) return
        if(!res[level]) res[level] = [];

        res[level].push(currNode.val)
        backtrack(currNode.left, level + 1)
        backtrack(currNode.right, level+ 1)
    }
    backtrack(root, 0)
    return res
}